styles/globals.css

```diff
@@ -0,0 +1,48 @@
+html {
+  font-size: 100%;
+  box-sizing: border-box;
+}
+
+*, *::before, *::after {
+  box-sizing: inherit;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  line-height: 1.3;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 1rem;
+  background-color: #fffff;
+}
+
+/* Text */
+
+a, a:visited, a:hover {
+  text-decoration: none;
+}
+
+img {
+  width: 1;
+}
+
+/* Flexbox */
+.flex {
+  display: flex;
+}
+
+.flex-jc-sb {
+    justify-content: space-between;
+  }
+
+.flex-jc-c {
+    justify-content: center;
+  }
+
+  .flex-jc-e {
```

Comment:
these util classes are great, but if you're working on a project with others it might be hard to adapt to. I'd just redo them to be more like boostrap. you would have a flex class (like you do) and then literally a `justify-content-center` class. It's more verbose but easier to read